README.md

```diff
@@ -6,15 +6,15 @@ This repository holds content related to the official Matroska specification and
 
 Local versions of the specification can be generated based on code in the `Makefile` directory and related dependencies. The dependencies required are `mmark`, `xml2rfc` and `xsltproc`. `mmark` is a Markdown processor written in Go, available [here](https://github.com/miekg/mmark) or, for Homebrew users, can be installed with `brew install mmark`. Installation instructions for `xml2rfc` (an XML-to-IETF-draft translator written in Python) are available on the [IETF Tools page](https://tools.ietf.org/tools/). `xsltproc` is a command line tool for applying XSLT stylesheets to XML documents. 
 
-To create local copies of the RFC in .txt, .md, and .html format, run `make`.
+To create local copies of the RFC in `.txt`, `.md`, and `.html` format, run `make`.
 
 To create a local copy of the website, run `make website`. This command has additional dependencies. See the "About this site" section below for installing and local rendering.
 
 To remove generated specifications, run `make clean`.
 
 ## About this site
 
-This site runs on [GitHub Pages](https://pages.github.com/) powered by [Jekyll](https://github.com/jekyll/jekyll/blob/master/README.markdown). Approved changes made to the `master` branch of this repository can be viewed instantly on the [associated webpage](http://matroska-org.github.io/matroska-specification/).
+This site runs on [GitHub Pages](https://pages.github.com/) powered by [Jekyll](https://github.com/jekyll/jekyll/blob/master/README.markdown). Approved changes made to the `master` branch of this repository can be viewed instantly on the [associated webpage](http://cellar-wg.github.io/matroska-specification/).
```

Comment:
I suggest to move to `https` at the same time.

Reply:
I agree.

Reply:
Done